.github/workflows/ci.yml

```diff
@@ -34,6 +34,7 @@ jobs:
           -x spotlessCheck
           -x checkstyleMain
           -x checkstyleTest
+          -x spotbugsMain
```

Comment:
It looks like this workflow may be causing the plugin to not install properly? Is this just a lint error or something with this specific workflow?

Reply:
The broken workflow was because of the linelinter, fix now!